bdk_core/src/chain_graph.rs

```diff
@@ -82,46 +82,53 @@ impl<I: ChainIndex> ChainGraph<I> {
             .determine_changeset(&update.chain)
             .map_err(UpdateFailure::Chain)?;
 
-        let conflicting_original_txids = update
+        let chain_conflicts = update
             .chain
             .iter_txids()
             // skip txids that already exist in the original chain (for efficiency)
             .filter(|&(_, txid)| self.chain.tx_index(*txid).is_none())
             // choose original txs that conflicts with the update
-            .flat_map(|(_, update_txid)| {
+            .flat_map(|(update_ind, update_txid)| {
                 let update_tx = update.graph.tx(*update_txid).expect("must exist");
                 self.graph
                     .conflicting_txids(update_tx)
-                    .filter_map(|(_, existing_txid)| {
-                        self.chain
-                            .tx_index(existing_txid)
-                            .map(|existing_index| (*update_txid, (existing_index, existing_txid)))
+                    .filter_map(move |(_, original_txid)| {
+                        self.chain.tx_index(original_txid).map(|original_ind| {
```

Comment:
we're using "existing" and "original" to mean the same thing here. Can we choose one. Or perhaps `conflicting_tx` would be better.

Reply:
Updated.